aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/CreateHandler.java

```diff
@@ -160,6 +166,22 @@ ApiVersion.DEFAULT, safeAddTags(this::createDbInstance)
                 .then(progress -> Commons.execOnce(progress, () ->
                                 updateAssociatedRoles(proxy, rdsProxyClient.defaultClient(), progress, Collections.emptyList(), desiredRoles),
                         CallbackContext::isUpdatedRoles, CallbackContext::setUpdatedRoles))
+                .then(progress -> Commons.execOnce(progress, () -> {
+                    if (ResourceModelHelper.shouldStartAutomaticBackupReplication(request.getPreviousResourceState(), request.getDesiredResourceState())
+                            && StringUtils.isNullOrEmpty(callbackContext.getDbInstanceArn())) {
+                        final DBInstance dbInstance = fetchDBInstance(rdsProxyClient.defaultClient(), progress.getResourceModel());
+                        callbackContext.setDbInstanceArn(dbInstance.dbInstanceArn());
+                    }
+                    return progress;
+                },  (m) -> !StringUtils.isNullOrEmpty(callbackContext.getDbInstanceArn()), (v, c) -> {}))
+                .then(progress -> Commons.execOnce(progress, () -> {
+                            if (ResourceModelHelper.shouldStartAutomaticBackupReplication(request.getPreviousResourceState(), request.getDesiredResourceState())) {
+                                return startAutomaticBackupReplicationInRegion(callbackContext.getDbInstanceArn(), proxy, progress, rdsProxyClient.defaultClient(),
+                                        ResourceModelHelper.getAutomaticBackupReplicationRegion(request.getDesiredResourceState()));
+                            }
+                            return progress;
+                        },
+                        CallbackContext::isAutomaticBackupReplicationStarted, CallbackContext::setAutomaticBackupReplicationStarted))
```

Comment:
There is no need for execOnce in this case because `startAutomaticBackupReplicationInRegion` starts a new execution chain. Uluru framework will make sure to exec this operation only once.